xml/tuning-ceph-tuning.xml

```diff
@@ -298,12 +298,25 @@ min_alloc_size_hdd=65536
 osd_memory_target = 6442450944
 osd_memory_cache_min = 4294967296
 </screen>
+     <tip>
+       <para>
+         As a best practice, adjust the memory by removing 16&nbsp;GB or 32&nbsp;GB
+         (depending on the OS), remove the MDS cache, and divide the remainder
+         by the number of OSDs on that host. Ensure you leave room for improvement.
```

Comment:
As a best practice, start with the full memory of the node. First, deduct 16 GB - 32 GB for the OS. Second, deduct appropriate amounts for any other workloads running on the node (e.g. MDS cache if MDS is colocated). Divide the remainder . . .